build/script.js

```diff
@@ -176,6 +177,8 @@ function createCompileTask(translation) {
                     conf.paths.bowerComponents, 'google-closure-library/closure/goog/base.js'),
               ])
               .pipe(gulpClosureCompiler(closureCompilerConfig))
+              .pipe(gulpReplace('BUILD_DASHBOARD_VERSION', conf.deploy.version.release))
```

Comment:
I'd export `BUILD_DASHBOARD_VERSION` constant to conf file and comment it properly. It should be clear what's it for.

Additionally I'd prefer to limit replace only to a single js file where those strings should be replaced and do it before the concatenation and minification. There is always a chance that someone will use such a string in some other place and it should not be replaced there.